### PR TITLE
Portfolio: add Contact form and finish mobile version

### PR DESCRIPTION
**Pull request by garikaipaidamoyo**
## In this milestone I: 🎯 
- created contact form and finished mobile version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -135,7 +135,26 @@ <h4 class="skills">Skills</h4>
...
         </div>
-    </section> 
+    </section>    
+    <section class="contact">
+        <fieldset class="contact-form">
+            <div class="background-img">
+                <h2 class="contact-heading">Contact me</h2>
+                <p class="contact-details">If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+                    I'd love to help with it!</p>
+            <form action="" method="Post" class="details-form">
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please make sure that your form uses the `action` and `method` attributes to integrate with [formspree service](https://formspree.io/html). The `action` value should be similar to `https://formspree.io/f/{form_id}` and have the `method` value equal `POST` :blush:

File: index.html [hunk 2]
```diff
@@ -135,7 +135,26 @@ <h4 class="skills">Skills</h4>
...
+                <input id="username" type="text" placeholder="Lorem Ipsum" required>
+                <input id="email" type="email" placeholder="Loremipsum@gmail.com" required>
+                <textarea id="textarea" placeholder="write your message..." required name="message" id="" cols="30" rows="10"></textarea>
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please make sure that the name input has the `maxlength="30"` attribute and the message input has the `maxlength="500"` attribute. :pray:

File: style.css [hunk 3]
```diff
@@ -240,3 +241,119 @@ hr {
...
+  text-align: center;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.51px;
+  border: 1px solid #fff;
+}
+
```
Comment (Owusu-Desmond) [inline style.css:352]: - [ ] Please make sure that your contact form button has interactions such as hover, active (when it's pressed), disabled, and enable (default state) styled as shown in the Figma template.

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @garikaipaidamoyo and @melkamu372 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (usorfaitheloho) [reply style.css:352]: You did well in implementing the button hover property, kindly use the design guideines for this so your design matches the design given 😄 

Figma design :
image

Your design :

https://github.com/garikaipaidamoyo/Portfolio/assets/20537712/aa4aed0d-8ad9-4ed5-b636-56775018b140

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @garikaipaidamoyo  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Implemented some of the required changes ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @garikaipaidamoyo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**